### ISSUE TITLE
V4: 如何单独配置tooptip中crosshairs线段的样式

### ISSUE THREAD

**Issue by 15802980231** [antvis/G2#5187]

**Comment by EmmaammE** [comment]
`crosshairs: {
          line: {
            style: {
              stroke: '#81848F',
              lineDash: [4, 2],
            },
          },
        },`

可以参考这个 https://github.com/antvis/G2/blob/v4.0.x/src/interface.ts#L1100

**Comment by 15802980231** [comment]
> `crosshairs: { line: { style: { stroke: '#81848F', lineDash: [4, 2], }, }, },`
> 
> 可以参考这个 https://github.com/antvis/G2/blob/v4.0.x/src/interface.ts#L1100

我的意思是怎样可以给xy轴线设置不同的样式

**Comment by EmmaammE** [comment]
> > `crosshairs: { line: { style: { stroke: '#81848F', lineDash: [4, 2], }, }, },`
> > 可以参考这个 https://github.com/antvis/G2/blob/v4.0.x/src/interface.ts#L1100
> 
> 我的意思是怎样可以给xy轴线设置不同的样式

继承Tooltip类覆盖renderCrosshairs方法？

**Comment by hustcc** [comment]
Sorry, due to problems in the program's architecture design, we are unable to effectively fix the issues. But it fixed in [G2 5.0](https://g2.antv.antgroup.com/zh/examples/general/line/#line-basic), you can have a try.

```ts
/**
 * A recreation of this demo: https://observablehq.com/@d3/line-chart
 */
import { Chart } from '@antv/g2';

const chart = new Chart({
  container: 'container',
  theme: 'classic',
  autoFit: true,
});

chart
  .line()
  .data({
    type: 'fetch',
    value:
      'https://gw.alipayobjects.com/os/bmw-prod/551d80c6-a6be-4f3c-a82a-abd739e12977.csv',
  })
  .encode('x', 'date')
  .encode('y', 'close');

chart.interaction('tooltip', {
  crosshairs: true,
  crosshairsStroke: 'red', // 👈🏻 here
})

chart.render();

```
